fem/datacollection.cpp

```diff
@@ -726,4 +726,394 @@ void VisItDataCollection::ParseVisItRootString(const std::string& json)
    }
 }
 
+
```

Comment:
Some comments on styling:
-  The mfem-way for commenting is `   // Close the data collection` or `   // close the data collection`,  not  `//close the data colection` .
- We leave 1 empty row between methods of the same class or for clarity inside functions.
- We leave 2 empty rows only between implementations of different classes.
- It is better to leave spaces around assignments: `myrank=0;` -> `myrank = 0;`.

Reply:
The code is modified.